vendor.mod

```diff
@@ -53,7 +53,7 @@ require (
 	github.com/hashicorp/serf v0.8.5
 	github.com/imdario/mergo v0.3.13
 	github.com/ishidawataru/sctp v0.0.0-20210707070123-9a39160e9062
-	github.com/klauspost/compress v1.15.12
+	github.com/klauspost/compress v1.16.3
```

Comment:
(File comments are great, now we just need commit comments.)

Your commit message for vendoring klauspost/compress should have https://github.com/klauspost/compress/compare/v1.15.12...v1.16.3 as the diff link.

Reply:
DOH! Fixed 👍